utils/mocks/halo.go

```diff
@@ -39,9 +39,10 @@ func HaloKeeperWithKeepers(_ testing.TB, account AccountKeeper, bank BankKeeper)
 		"uusdc",
 		account,
 		nil,
+		registry,
 	)
 
-	bank = bank.WithSendCoinsRestriction(k.SendRestrictionFn)
+	bank = bank.WithSendCoinsRestriction(bank.Restriction)
```

Comment:
This change should be reverted, because this simply sets the send restriction to nothing 😄

```suggestion
	bank = bank.WithSendCoinsRestriction(k.SendRestrictionFn)
```